ruby_connectors/spec/temporal/workflows/connection_status_workflow_spec.rb

```
@@ -1,20 +1,23 @@
-# frozen_string_literal: true
+# convert this tests to integration tests and use vcr
```

rubocop

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)